database/migrations/20230818100709_create_users_table.sql

```diff
@@ -0,0 +1,16 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE "public"."users" (
+    id uuid DEFAULT uuid_generate_v1(),
+    "email" text UNIQUE,
```

Comment:
```suggestion
    "email" citext UNIQUE,
```

I recommend using `citext` (case insensitive text) for emails so that the UNIQUE conditions will not allow `MARK@g.com` when there is already `mark@g.com` in the DB 👍

Reply:
@malparty 🙏 
```
ERROR: type \"citext\" does not exist (SQLSTATE 42704)"
```

Reply:
Unresolved. https://github.com/markgravity/golang-ic/pull/38/files#r1299689157 can help ℹ️

Reply:
I insist on this one, as it's an important concern for any web applications. You don't want to end up with 2 different users using the same email 🤯

Reply:
@malparty I updated it here 🙏 
[6f07328](https://github.com/markgravity/golang-ic/pull/38/commits/6f0732868f1411ea687af2420ed6d260e6b026b8)